pkg/demo/demo.go

```diff
@@ -174,6 +230,24 @@ AND (
 	return body.String(), nil
 }
 
+// condition wraps the output of WherePart inside "AND (...)".
+// We "know" this condition will not be the first condition in the query.
+func condition(col string, args []string) (string, error) {
```

Comment:
additionalCondition?

Reply:
or extraCondition? (or something)

Reply:
everybody else is better at names than I am :)
fixed

Reply:
👍